Makefile

```diff
@@ -231,6 +231,8 @@ CODEGEN_ROOT = $(shell $(GO_CMD) env GOMODCACHE)/k8s.io/code-generator@$(CODEGEN
 code-generator:
 	@GOBIN=$(PROJECT_DIR)/bin GO111MODULE=on $(GO_CMD) install k8s.io/code-generator/cmd/client-gen@$(CODEGEN_VERSION)
 	cp -f $(CODEGEN_ROOT)/generate-groups.sh $(PROJECT_DIR)/bin/
+	cp -f $(CODEGEN_ROOT)/generate-internal-groups.sh $(PROJECT_DIR)/bin/
```

Comment:
why did running `generate-internal-groups.sh` become necessary for 1.28?

Reply:
idk but it fails without it.  

See https://github.com/kubernetes-sigs/kueue/pull/1133

Reply:
It is deprecated actually so maybe it won't follow normal behavior. :(